owncloud/core
# support for AD primary groups

With this PR ownCloud reads the primaryGroupID from a user and fetches the primary group on requests for the user's groups. Also, when a group users are retrieved ownCloud will look for users who have this group as primary group. This is necessary, because primary groups are not represented by AD as ordinary groups.

How to test for **AD users**:
- see on the Users page whether the primary group is visible for the users now

Hint for **other LDAP server users**:
- the attribute is attempted to be read independent of the LDAP server  in use (we do not attempt to detect it, anyway). 

This fixes #7052

Please test/review @deajan @Xenopathic or others
